CL/cl_platform.h

```diff
@@ -53,80 +53,80 @@ extern "C" {
 #define CL_API_PREFIX_COMMON CL_API_PREFIX_USER
 
 #define CL_API_SUFFIX__VERSION_1_0 CL_API_SUFFIX_COMMON
-#define CL_EXT_SUFFIX__VERSION_1_0 CL_API_SUFFIX_COMMON
+#define CL_API_SUFFIX__VERSION_1_0 CL_API_SUFFIX_COMMON
```

Comment:
Looks like a duplicate definition here and for the other versions below.

Reply:
Good catch. It was a bit more than a sed job :).

Reply:
Now fixed.